pkg/bff/members.go

```diff
@@ -122,12 +140,50 @@ func (s *Server) Overview(c *gin.Context) {
 		out.TestNet.Vasps = int(testnet.Vasps)
 		out.TestNet.CertificatesIssued = int(testnet.CertificatesIssued)
 		out.TestNet.NewMembers = int(testnet.NewMembers)
+
+		if testnetID != "" {
+			// Check if we received the VASP details from the testnet
+			if testnet.Vasp == nil {
+				log.Error().Msg("expected VASP details from testnet Summary RPC")
+				c.JSON(http.StatusInternalServerError, api.ErrorResponse(fmt.Errorf("could not retrieve testnet VASP details")))
+				return
+			}
+
+			out.TestNet.MemberDetails = api.MemberDetails{
+				ID:          testnet.Vasp.Id,
+				Status:      testnet.Vasp.Status.String(),
+				CountryCode: testnet.Vasp.Country,
+			}
+		} else {
+			out.TestNet.MemberDetails = api.MemberDetails{
+				Status: pb.VerificationState_NO_VERIFICATION.String(),
+			}
+		}
 	}
 
 	if mainnet != nil {
 		out.MainNet.Vasps = int(mainnet.Vasps)
 		out.MainNet.CertificatesIssued = int(mainnet.CertificatesIssued)
 		out.MainNet.NewMembers = int(mainnet.NewMembers)
+
+		if mainnetID != "" {
+			// Check if we received the VASP details from the mainnet
+			if mainnet.Vasp == nil {
+				log.Error().Msg("could not retrieve mainnet VASP details")
+				c.JSON(http.StatusInternalServerError, api.ErrorResponse(fmt.Errorf("could not retrieve mainnet VASP details")))
+				return
+			}
+
+			out.MainNet.MemberDetails = api.MemberDetails{
+				ID:          mainnet.Vasp.Id,
+				Status:      mainnet.Vasp.Status.String(),
+				CountryCode: mainnet.Vasp.Country,
```

Comment:
We could return more of the members information we get from `Summary`, but this is just what was originally defined for the overview endpoint.